### DISCUSSION TITLE
Add (including closed issues) to issue template

### DISCUSSION THREAD

**Discussion by SteveAmor** [InfiniTimeOrg/InfiniTime#1796] [Ideas]
Looks like I can't raise a PR to suggest the following addition to the Issues Template:

**Verification**
I searched for similar bug reports (including closed issues) and found none was relevant

**Comment by Elements6007** [comment]
does editing [.github/ISSUE_TEMPLATE/bug-report.yaml](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/.github/ISSUE_TEMPLATE/bug-report.yaml) and open a PR not work?

**Comment by SteveAmor** [comment]
@Elements6007 thank you. My googlefoo failed me (advising to go to the github repo settings) but this does indeed look like the solution to being able raise a PR which I will do soon and then close this discussion.

**Comment by SteveAmor** [comment]
PRs raised. #1816 #1817
